index.js

```diff
@@ -13,8 +13,17 @@ app.use(staticMiddleware)
 app.use(jsonParser)
 
 app.post('/tasks', (req, res) => {
-  addToTasks(req.body)
-  res.sendStatus(200)
+  knexFunction.addToTasks(req.body)
```

Comment:
Instead of naming this object `knexFunction`, you should name it `tasks`. And you can rename the methods. _e.g._ `tasks.add` `tasks.list`